### PR TITLE
Gpt: add profile and update evaluator

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: Evaluator/TA/ai_evaluator/ai.py [hunk 1]
```diff
@@ -43,7 +43,6 @@ class GPTEvaluator(evaluators.TAEvaluator):
         "Kaufman Adaptive Moving Average": tulipy.kama,
         "Hull Moving Average": tulipy.kama,
         "RSI: Relative Strength Index": tulipy.rsi,
-        "Stochastic RSI": tulipy.stochrsi,
```
Comment (Herklos) [inline Evaluator/TA/ai_evaluator/ai.py:46]: 😲 it's not working ?

File: profiles/gpt_daily_trading/specific_config/TechnicalAnalysisStrategyEvaluator.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    "default_config": [
+        "DoubleMovingAverageTrendEvaluator",
+        "RSIMomentumEvaluator"
+    ],
+    "required_evaluators": [
+        "*"
+    ],
+    "required_time_frames": [
+        "4h"
```
Comment (Herklos) [inline profiles/gpt_daily_trading/specific_config/TechnicalAnalysisStrategyEvaluator.json:14]: 👍

File: Evaluator/TA/ai_evaluator/ai.py (hunk 1 above)
Comment (GuillaumeDSM) [reply Evaluator/TA/ai_evaluator/ai.py:46]: it's giving results that are not allowing to forcast a trend

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**